### ISSUE TITLE
Model name clarification

### ISSUE THREAD

**Issue by Erol444** [AVAuco/deeparuco#3]
Hi! I'm interested in porting the models to Edge Inferencing (specifically to OAK cameras), and there are quite a few `.pt` files in `models` folder.
Looking at your paper, I assume the `_n` at the end of the model name stands for Yolo nano and `_s` stands for Yolo small, but what does the `_b_` / `_bc_` /  `_bn_` / `_bnc_` in the middle of the model name stand for?

```
det_luma_b_n.pt
det_luma_b_s.pt
det_luma_bc_n.pt
det_luma_bc_s.pt
det_luma_bn_n.pt
det_luma_bn_s.pt
det_luma_bnc_n.pt
det_luma_bnc_s.pt
```

**Comment by rafabs97** [comment]
Thank you for your interest in our work! The string in the middle of each model name represents the data augmentation techniques applied to the base FlyingArUco v2 dataset before training that model. Specifically, we used combinations of blur (`_b_`) with noise (`_n_`), and/or color shift (`_c_`). Through our experiments, we determined that combining blur and color shift (`_bc_`) yielded the best results on our test dataset; however, we have included all the models we trained in this repository, as users may find them more useful for their specific needs.
